src/Oscoin/Configuration/Options.hs

```diff
@@ -0,0 +1,25 @@
+module Oscoin.Configuration.Options
+    ( environmentParser
+    )
+where
+
+import           Oscoin.Prelude hiding (option)
+
+import           Oscoin.Configuration
+                 ( Environment(Development)
+                 , allEnvironments
+                 , readEnvironment
+                 , renderEnvironment
+                 )
+
+import           Options.Applicative
+
+
+environmentParser :: Parser Environment
+environmentParser = option (eitherReader readEnvironment)
```

Comment:
Blue paint bikeshedding: Feels slightly artificial to have a `Oscoin.Configuration.Options` module exporting only a `Parser` -- perhaps this belongs in one of those `CLI/Parser/Whatever` submodules?

Reply:
This was accidentally committed - the parser is in `Oscoin.Configuration`.

Also consider the `CLI` is about the (administrative) commands, so the reason to have them in the "gobal" configuration is to be able to re-use between `Node` and `CLI` options.